bridge/generic-queue/src/gcp_pubsub.rs

```diff
@@ -186,6 +186,10 @@ impl<T: DeserializeOwned + Send + Serialize + Sync> Delivery<T> for GCPPubSubDel
         serde_json::from_slice(&self.message.message.data).map_err(Into::into)
     }
 
+    fn raw_payload(&self) -> Result<String, QueueError> {
```

Comment:
```suggestion
    fn raw_payload<'a>(&'a self) -> Result<&'a str, QueueError> { ... }
```

Could we get away with returning a `&str`, using something like https://doc.rust-lang.org/std/str/fn.from_utf8.html?

That way, we give the consumers the option of `.clone()`ing if and only if they need to.

Reply:
I think this might need to be a Cow, for cases where we only have raw bytes for the payload, but otherwise I think you're right.

Reply:
Derp. You are 100% correct. The sad part is we will need to have an owned value to talk to the JS Runtime, but we can borrow right up until a certain point.

I haven't refactored `TransformerInput` to be lifetime'd, but it's possible we could avoid the allocation there, deferring all the way up to the actual script execution.

I'll look at that in a separate PR if time allows.